build.gradle

```diff
@@ -24,7 +24,7 @@ apply plugin: 'application'
 apply plugin: 'java'
 apply plugin: 'org.springframework.boot'
 apply plugin: 'io.spring.dependency-management'
-apply from: rootProject.file("gradle/install-git-hooks.gradle")
```

Comment:
Is this on purpose

Reply:
Yes see https://github.com/jua-cs/jua/pull/8/commits/c5ca41558bcf6d51230ea530f52bc8efe2367648 for more details

Reply:
great job!

Reply:
most important change of the PR cc @sfluor

Reply:
I think we can leave it it's not automatic you have a o explicitly run installGitHooks to install them

Reply:
No, each `gradle build` makes it reappears ! :angry: